collector/src/framework/controllers/repositories_controller.ts

```diff
@@ -1,10 +1,18 @@
 import express from 'express';
 import axios from 'axios';
 import yaml from 'js-yaml';
-import { SystemModel } from '../../framework/database/schema';
 
 import { RegisterSystem } from '../../core/usecases/register_system';
 
+class SystemModel {
+  constructor(private readonly sys: any) {}
+
+  save(): Promise<any> {
+    console.log('system saved');
+    return new Promise((resolve) => resolve(this.sys));
+  }
+}
+
```

Comment:
Isso é algum vestígio esquecido? Se não for, tem cara de que tá no lugar errado.

Reply:
Essa classe eu criei meio suja mesmo porque lá na `RegisterSystem.run()' a gente cria uma instancia desse SystemModel e chama um método save dela. Tecnicamente o Bruno vai mudar isso na parte dele, então eu preferi criar essa classe meio que mockada mesmo só pra não modificar nada do que ele vai mexer lá, mas confesso que fiquei um pouco na dúvida se precisava ou não disso. Qual a melhor solução nesse caso?

Reply:
Não sei se eu entendi direito a origem dessa classe...
Quando tivermos call, a gente fala sobre isso, pode ser? Fica na pilha